public/pages/workflow_detail/component_details/input_fields/select_field.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import {
+  EuiFormRow,
+  EuiSuperSelect,
+  EuiSuperSelectOption,
+  EuiText,
+} from '@elastic/eui';
+import { Field, FieldProps, useFormikContext } from 'formik';
+import {
+  IComponentField,
+  WorkspaceFormValues,
+  getInitialValue,
+  isFieldInvalid,
+} from '../../../../../common';
+
+// TODO: Should be fetched from global state.
+// Need to have a way to determine where to fetch this dynamic data.
+const existingIndices = [
```

Comment:
I know this will change to some fetch for all indices as you mention in comment but the select field shouldn't be specific to indices right? Just "existing options"? We can have a list of model IDs here too

Reply:
Correct, just have hardcoded values for testing functionality. Has nothing to do with indices.